src/rgw/rgw_sal_motr.cc

```diff
@@ -1077,9 +1077,11 @@ int MotrBucket::list_multiparts(const DoutPrefixProvider *dpp,
   int ocount = 0;
   rgw_obj_key last_obj_key;
   *is_truncated = false;
-  for (const auto& bl: val_vec) {
+
+   for (auto& bl : val_vec) {
+    
```

Comment:
Why? 1st, indentation is broken. 2nd - why `const` is removed?

Reply:
Fixed